### PR TITLE
Update increase-the-recoverable-quota-for-mailboxes-on-hold.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the URL for the deleted item retention period.

Fixes#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10431

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/increase-the-recoverable-quota-for-mailboxes-on-hold.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ The first step is to create a custom retention tag (called a retention policy ta
...
+    > We recommend that the retention period (defined by the  _AgeLimitForRetention_ parameter) for the Recoverable Items RPT is the same as the deleted item retention period for the mailboxes that the RPT will be applied to. This allows a user the entire deleted item retention period to recover deleted items before they are moved to the archive mailbox. In the previous example, the retention period was set to 30 days based on the assumption that the deleted item retention period for mailboxes is also 30 days. An Exchange Online mailbox is configured to retain deleted items for 14 days, by default. But you can change this setting to a maximum of 30 days. For more information, see [Change the deleted item retention period for a mailbox in Exchange Online](/exchange/recipients-in-exchange-online/manage-user-mailboxes/change-deleted-item-retention).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/increase-the-recoverable-quota-for-mailboxes-on-hold.md]:
```suggestion
    > We recommend that the retention period (defined by the _AgeLimitForRetention_ parameter) for the Recoverable Items RPT is the same as the deleted item retention period for the mailboxes that the RPT will be applied to. This allows a user the entire deleted item retention period to recover deleted items before they are moved to the archive mailbox. In the previous example, the retention period was set to 30 days based on the assumption that the deleted item retention period for mailboxes is also 30 days. An Exchange Online mailbox is configured to retain deleted items for 14 days, by default. But you can change this setting to a maximum of 30 days. For more information, see [Change the deleted item retention period for a mailbox in Exchange Online](/exchange/recipients-in-exchange-online/manage-user-mailboxes/change-deleted-item-retention).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@robmazz Please merge this PR. Thanks

**Comment by robmazz** [comment]
@msbemba - thanks, updated in separate PR.

**Pull request closed without merging**